bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
 - shell: git rev-parse --show-toplevel 2>/dev/null
   register: toplevel
 
+- name: Ensure that the apt cache is up to date
+  apt:
+    update_cache: yes
```

Comment:
A few lines down, we have an identical `apt` task that updates the cache. So what is the reason for adding this additional cache-updating task?